kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md

```diff
@@ -0,0 +1,341 @@
+# WebLogic Sample Domain Home in Docker Image
+
+The sample scripts demonstrate the creation of a WebLogic domain home in docker image. The scripts also generate the domain custom resource YAML file, which can then be used to start the Kubernetes artifacts of the corresponding domain. Optionally, the scripts start up the domain custom resource, and WebLogic Server pods and services.
```

Comment:
Docker is always capitalized.

I think the the domain custom resource YAML file is being renamed to the domain resource YAML file and domain custom resource will be referred to as the domain resource. Please check if I am correct or not about this.